New Home Page

robbiejaeger opened the pull request:
Experimenting with a new look for the FE-curriculum home page.

Here is a WIP of a splash image for the front page. I'm still torn on what to do for the remaining links needed on that page:
* Calendar (still need a link to this since it's in the top nav?)
* Lessons (still need a link to this since it's in the top nav?)
* Projects (still need a link to this since it's in the top nav?)

* Professional Development
* Mentor and Posse Info
* Pairing Sign-Up Sheet
* Job Search Resources
* Mod 0 Capstone 

Would love some direction or to see some spikes on this. How do we put the links on the page in a nice way?

This branch also contains some general cleanup of unused items in the curriculum site.

---

*Desktop View:*
![Desktop](https://user-images.githubusercontent.com/17582916/54561897-6a0c8d00-498b-11e9-83a1-1254e99c997c.png)

--- 

*Mobile View:*
![Mobile](https://user-images.githubusercontent.com/17582916/54561924-7d1f5d00-498b-11e9-8eed-b56ca1d91d67.png)

letakeane commented:
Maybe for the remaining links, the three sections (Daily Turing, Your Resources, Before & After Turing) could be turned into columns, with the buttons arrayed vertically?

Although Before & After Turing links aren't especially high priority and could be "below the fold" ....

I think the Calendar, Lessons, and Projects links should still be there, even though they're also in the top nav.

brittanystoroz commented:
Ooooh looks fancy

brittanystoroz commented on _sass/home-page.scss:
```diff
@@ -0,0 +1,75 @@
+
+@media (max-width: $tablet-width) {
+  .splash {
+    height: 350px;
```
I think this might look better at 400px, what do you think?

robbiejaeger commented:
I’ve added some additional sections to the page to include some links to resources. Links I have removed to simplify and reduce unnecessary clutter on the page:

* Mentor and Posse Info - no longer used because of mentor assignments
* Pairing Sign-Up Sheet - no longer used because of Paired app
* Job Search Resources - link outdated and not used
* Mod 0 Capstone - this is now it’s own website maintained by someone else, not completely sure yet

I also added a short section featuring the Paired app - seems to be well maintained and used by students.

![image](https://user-images.githubusercontent.com/17582916/60854420-75472b00-a1bd-11e9-9106-bbbe314a522e.png)
![image](https://user-images.githubusercontent.com/17582916/60854444-842ddd80-a1bd-11e9-909a-87d3867cc9f1.png)

ameseee commented:
I think this looks great @robbiejaeger! 

What do you think about using the same `lighten($turing-secondary, 25%);` on the hover/active states for the nav items, same color as the .`lesson-segment` boxes?

![image](https://user-images.githubusercontent.com/25447342/60890211-182c9300-a218-11e9-983c-195100ed0d79.png)

robbiejaeger commented:
Added the mentor list back in because Kayt said it’s still used. Also updated the version of jQuery we use vie the CDN link.

![image](https://user-images.githubusercontent.com/17582916/60907369-9435d380-a236-11e9-9253-64bc51eff2b5.png)
![image](https://user-images.githubusercontent.com/17582916/60907385-a283ef80-a236-11e9-9442-99495d44195a.png)

robbiejaeger commented:
Louisa gave the feedback that the “Paired” section is really big. Do we want to feature it this much? We can probable make this more subtle in future iterations, but I reduced the image size in the meantime.
